app/components/ui-table/cell/events/view/videoroom/cell-video-url.js

```diff
@@ -1,5 +1,15 @@
 import classic from 'ember-classic-decorator';
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
 @classic
-export default class CellVideoUrl extends Component {}
+export default class CellVideoUrl extends Component {
+  @computed
+  get shortUrl() {
+    let url = this.record.replace('https://', '').replace('www.', '');
+    if (url.length > 37) {
+      url = url.slice(0, 36) + ' . . .';
```

Comment:
It'll be better if you use CSS for truncating

https://daily-dev-tips.com/posts/css-truncate-text-with-ellipsis/

There must be a class in semantic UI already